vendor/vendor.json

```diff
@@ -922,10 +928,10 @@
 			"revisionTime": "2017-06-05T02:18:34Z"
 		},
 		{
-			"checksumSHA1": "QHuUJtX9TiA32UXIhsQSe6x4qVA=",
```

Comment:
Ah, they finally added that api. I'll be changing the vault wrapper package to use that as well once this is merged